### ISSUE TITLE
Add style for cancelled task in tasks syntax (PlainTasks syntax)

### ISSUE THREAD

**Issue by manoverda** [equinusocio/material-theme#536]
Using your color scheme with _PlainTasks_ package is just awesome! But it would be perfect with some improvements. The most important is the support of **cancelled tasks**. **Bold** and _Italic_ could also be improved to look like in _Markdown_. Here a preview with `Material-Theme-Darker` color scheme and `tasks-eighties-colored` color scheme:

![cancelledtasks](https://cloud.githubusercontent.com/assets/8159138/11681376/e6479012-9e5e-11e5-8801-a8744a293561.png)
![cancelledtasks2](https://cloud.githubusercontent.com/assets/8159138/11681377/e64985ac-9e5e-11e5-8063-df9c51719f29.png)

**Comment by manoverda** [comment]
Corrected it by adding `text.cancelled` scope in Comments section:
``` XML
    <dict>
      <key>name</key>
      <string>Comments</string>
      <key>scope</key>
      <string>comment, punctuation.definition.comment, text.cancelled</string>
      <key>settings</key>
      <dict>
        <key>foreground</key>
        <string>#444444</string>
      </dict>
    </dict>
```

I also add a notes section to display notes in tasks in different way than tasks themselves:
``` XML
    <dict>
      <key>name</key>
      <string>notes</string>
      <key>scope</key>
      <string>notes</string>
      <key>settings</key>
      <dict>
        <key>foreground</key>
        <string>#8EACE3</string>
      </dict>
    </dict>
```
Before:
![before](https://cloud.githubusercontent.com/assets/8159138/11712925/5fbdb9e0-9f30-11e5-843f-1dd276a17669.png)
After:
![after](https://cloud.githubusercontent.com/assets/8159138/11712927/630fd7b8-9f30-11e5-976e-408d87373723.png)

Didn't manage the __Bold__ and _Italic_ words yet.

**Comment by manoverda** [comment]
OK, now that I understood how it works, I managed the bold and italic words:
Added `todo.bold` scope in Bold section and `todo.italic` in Italic section

**Comment by equinusocio** [comment]
@manoverda you will make PR? :)

**Comment by manoverda** [comment]
Sure. My first one :)
I think I have to change all the color schemes of the theme in the same way?
I can't use github at work, so I'll try to do that this week-end at home.

**Comment by equinusocio** [comment]
Yes, you have to fix also other schemes. Maybe for the lighter scheme you have to change the colors to keep contrast wi the white background.

**Comment by manoverda** [comment]
No problem. I'll try to keep in mind the colors used in each theme to be compliant with.

**Comment by manoverda** [comment]
I sent a PR #550 for this issue.

**Comment by diego898** [comment]
Hello! How can I use Materia's color scheme with plaintasks? It seems like this PR fixed the issue, but when I have the following in my plaintasks user settings file:

```
{
    "color_scheme": "Packages/Material Theme/schemes/Material-Theme.tmTheme",   // From Material Theme
}
```

I don't really see any built in support:
![screen shot 2016-10-03 at 9 45 44 am](https://cloud.githubusercontent.com/assets/742407/19045530/32830a9e-894e-11e6-8d5f-528063e966c3.png)

**Comment by manoverda** [comment]
Hello.
Unfortunately, the task syntax was removed with a big syntax update.
You can find [here](https://github.com/manoverda/material-theme/commit/7aa2cf66089cd5ac7f567796563a987793b53f23) what I did in order to support the task syntax. I only did it for Dark syntax, so if you want it for other themes, you shall port it to the theme you want and adapt the colors.

Regards.

**Comment by equinusocio** [comment]
Fixed from 4.0.0
